Carthage/Checkouts/Commandant

```diff
@@ -1 +1 @@
-Subproject commit fdb18a2602fcf6641faad87b4a1667995a68b706
+Subproject commit 186fc8d7937cbdced7bc9e90114079241d864f77
```

Comment:
It looks like you accidentally reset the Commandant submodule during a merge/rebase. Can you undo this change?